db/dataset.py

```diff
@@ -315,3 +315,27 @@ def _delete_snapshots_for_dataset(connection, dataset_id):
         DELETE FROM dataset_snapshot
               WHERE dataset_id = :dataset_id""")
     connection.execute(query, {"dataset_id": dataset_id})
+
+
+def add_recordings(dataset, dataset_id):
+    """Adds new recordings to a class in a dataset"""
+
+    with db.engine.begin() as connection:
+        result = connection.execute("""SELECT id FROM dataset_class WHERE name = %s and dataset = %s""",
+                                    (dataset["class_name"], dataset_id))
+        clsid = result.fetchone()
```

Comment:
If there is no class with this name, then `clsid` will be `None`. You need to check this here before you use it to insert the new recording ids